src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -235,12 +235,16 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode)
                 ->arrayNode('workflows')
```

Comment:
missing ``->fixXmlConfig('workflow')`` on the root node IMO, as having a ``workflows`` node for the list of workflows does not match the way things are done (and btw, it will cause issues when you define 1 vs several workflows in XML currently AFAIK. Please test this case)